CMakeLists.txt

```diff
@@ -27,7 +27,28 @@ set(CMAKE_MODULE_LINKER_FLAGS_INIT "-rdynamic -Wl,--export-dynamic -Wl,--enable-
 set(CMAKE_MODULE_LINKER_FLAGS_RELEASE_INIT "-Wl,--gc-sections -Wl,-z,combreloc")
 set(CMAKE_SHARED_LINKER_FLAGS_INIT "-rdynamic -Wl,--export-dynamic -Wl,--enable-new-dtags -Wl,--as-needed -Wl,-z,defs")
 set(CMAKE_SHARED_LINKER_FLAGS_RELEASE_INIT "-Wl,--gc-sections -Wl,-z,combreloc")
-include(IrodsRunpathDefaults)
+if (IRODS_ENABLE_430_COMPATIBILITY)
+  if (NOT DEFINED CMAKE_SKIP_BUILD_RPATH)
```

Comment:
Why do we need to check if each property is defined before setting them?

Reply:
Because we don't want to override values passed in via command line

Reply:
Got it.